src/components/policy/policy_external/test/sql_pt_representation_test.cc

```diff
@@ -1633,231 +1633,6 @@ TEST(SQLPTRepresentationTest3, RemoveDB_RemoveDB_ExpectFileDeleted) {
   EXPECT_FALSE(file_system::FileExists(path));
 }
 
-TEST_F(SQLPTRepresentationTest,
```

Comment:
@ychernysheva if you want to remove the disabled unit tests, you should create a new ones (or make sure it exists) which cover the test cases which supposed to be covered by the removed UT

Reply:
I have added few unit tests to check Save(..) and GenerateSnapshot() methods, which works with current version of preloaded pt. Removed tests cannot work because they use hard-coded values, which now don't correspond with current state of preloaded pt and don't map with existing sections of this table.
So I can say that I check correctness of work of this methods for module meta section. In the future we may add other similar tests for other sections, but not in the scope of this feature, i think. Deleted tests in fact don't work now and cannot work, so why we need to keep them?

Reply:
I propose to move changes related to the tests removing to a separate commit and create a detailed explanation of why they should be removed